### PR TITLE
New package: iMobieInc.DroidKit version 1.0.0.9

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/118744)

### CODE REVIEW

**Comment by stephengillie** [comment]
Should this be a portable package?

File: manifests/i/iMobieInc/DroidKit/1.0.0.9/iMobieInc.DroidKit.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: iMobieInc.DroidKit
+PackageVersion: 1.0.0.9
+Installers:
+- InstallerUrl: https://www.imobie.com/go/download.php?product=dk&id=2
```
Comment (Trenly) [inline manifests/i/iMobieInc/DroidKit/1.0.0.9/iMobieInc.DroidKit.installer.yaml:7]:
```suggestion
- InstallerUrl: https://dl.imobie.com/droidkit-en-setup.exe
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by upintheairsheep** [comment]
> Hello @upintheairsheep,
> 
> I am sorry to report that the Sha256 Hash does not match the installer. This may be caused by using a vanity URL rather than a URL directly to the binary.
> 
> Please check the HASH and submit again. Sorry for the inconvenience.
> 
> Template: msftbot/validationError/installers/hashMismatch

An update has occurred.

**Comment by stephengillie** [comment]
> Specified hash doesn't match.
Actual hash:   01F79B92DA6232CBB7580F32FB6F2379AB29DCB678489ADDE6FE5DACF77866CB
Expected hash: e6b021728b28398bef49bd25831d4b1824014136b16393982fdaf3997198b320

File: manifests/i/iMobieInc/DroidKit/1.0.0.9/iMobieInc.DroidKit.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: iMobieInc.DroidKit
+PackageVersion: 1.0.0.9
+Installers:
+- InstallerUrl: https://dl.imobie.com/droidkit-en-setup.exe
+  Architecture: x86
+  InstallerType: nullsoft
+  InstallerSha256: E6B021728B28398BEF49BD25831D4B1824014136B16393982FDAF3997198B320
```
Comment (stephengillie) [inline manifests/i/iMobieInc/DroidKit/1.0.0.9/iMobieInc.DroidKit.installer.yaml:10]:
```suggestion
  InstallerSha256: 01F79B92DA6232CBB7580F32FB6F2379AB29DCB678489ADDE6FE5DACF77866CB
```

**Comment by upintheairsheep** [comment]
keepalive

**Comment by stephengillie** [comment]
> Installer Verification Analysis Context Information: 
Specified hash doesn't match.
Actual hash:   AE2001D5B60A2F0BD8E72C0106363950CD9F68E9CE42B9A40B0AF26814908809
Expected hash: 01f79b92da6232cbb7580f32fb6f2379ab29dcb678489adde6fe5dacf77866cb

File: manifests/i/iMobieInc/DroidKit/1.0.0.9/iMobieInc.DroidKit.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: iMobieInc.DroidKit
+PackageVersion: 1.0.0.9
+Installers:
+- InstallerUrl: https://dl.imobie.com/droidkit-en-setup.exe
+  Architecture: x86
+  InstallerType: nullsoft
+  InstallerSha256: 01F79B92DA6232CBB7580F32FB6F2379AB29DCB678489ADDE6FE5DACF77866CB
```
Comment (stephengillie) [inline manifests/i/iMobieInc/DroidKit/1.0.0.9/iMobieInc.DroidKit.installer.yaml:10]:
```suggestion
  InstallerSha256: AE2001D5B60A2F0BD8E72C0106363950CD9F68E9CE42B9A40B0AF26814908809
```

**Comment by stephengillie** [comment]
I couldn't find any files that were modified by this package. Is this expected?

**Pull request closed without merging**